lambda/update/main.go

```diff
@@ -82,39 +91,9 @@ func (l *Lambda) HandleEvent(ctx context.Context, event events.APIGatewayProxyRe
 	return response, nil
 }
 
-func applyUpdate(lpa *shared.Lpa, update shared.Update) ([]shared.FieldError, error) {
-	validationErrs := []shared.FieldError{}
-
-	for index, change := range update.Changes {
-		pointer, err := jsonpointer.New(change.Key)
-		if err != nil {
-			return validationErrs, err
-		}
-
-		current, _, err := pointer.Get(*lpa)
-		if err != nil {
-			return validationErrs, err
-		}
-
-		if current != change.Old {
-			validationErrs = append(validationErrs, shared.FieldError{
-				Source: fmt.Sprintf("/changes/%d/old", index),
-				Detail: "does not match existing value",
-			})
-		}
-
-		_, err = pointer.Set(lpa, change.New)
```

Comment:
this would need to know the type it is going to set (if you want to see why the current one breaks, try setting `/signedAt`), so I've removed in favour of having a specific `Apply` object that does know what the types are

Reply:
Makes sense, thanks for reworking that